aospy/test/test_calc_basic.py

```diff
@@ -160,6 +161,45 @@ def setUp(self):
         }
 
 
+test_params = {
+    'proj': example_proj,
+    'model': example_model,
+    'run': example_run,
+    'var': var_not_time_defined,
+    'date_range': 'default',
+    'intvl_out': 1,
+}
+
+
+@pytest.mark.parametrize(('dtype_out_time'), [None, 'None', []])
```

Comment:
I think you can remove the parentheses around `'dtype_out_time'` here.  Also, is there a reason we are testing with the value `'None'`?